.github/workflows/ci.yml

```diff
@@ -9,12 +9,12 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
-    - name: Set up Ruby 2.6
-      uses: actions/setup-ruby@v1
```

Comment:
Why this rollback? what is wrong in ruby 2.6?

Reply:
It cannot be found by the respective GHA. Also, I tested it on all rubies up to 3.2.